### PR TITLE
Adding translation for certain pages

### PR DESCRIPTION
**Pull request by imran1289-ah**
This PR adds the translation for the following pages. Keep in mind that only static text is translated, so anything dynamic or is part of something dynamic is not translated. Since a lot of the pages are being worked on currenlty (notably to pages related to jobs), i will need your guys help to translate the additional pages and new pages you guys are adding
- sign in page
- sign up page
- navbar
- user profile
- public user profile
- user timeline
- search page
- waiting connections
- chat page

### CODE REVIEW

**Comment by imran1289-ah** [comment]
I have no idea what this Vercel thing is doing lol

**Comment by DasDeveloper** [comment]
Unable to change languages once we go to the user profile page.
Screen Shot 2023-03-27 at 4 00 39 PM

**Comment by DasDeveloper** [comment]
Since these words dont appear dynamically, we can translate them in the jobs applied page. 

Screen Shot 2023-03-27 at 4 05 17 PM

**Comment by DasDeveloper** [comment]
Overall, looks good. Good job.

File: src/client/src/components/PublicUserProfile.jsx [hunk 1]
```diff
@@ -299,7 +307,7 @@ const PublicUserProfile = () => {
                 })
               ) : (
                 <p className="userBio">
-                  The user does not have any connections
+                  {t("The user does not have any connections")}
                 </p>
               )}
             </div>
```
Comment (DasDeveloper) [inline src/client/src/components/PublicUserProfile.jsx:313]: Missing footer where user can change to their language of choice.

**Review by DasDeveloper — APPROVED** [review]

**Comment by imran1289-ah** [comment]
> Unable to change languages once we go to the user profile page. Screen Shot 2023-03-27 at 4 00 39 PM

Added the footer in the profile pages

**Comment by imran1289-ah** [comment]
> Since these words dont appear dynamically, we can translate them in the jobs applied page.

Added the translation for this page
image

**Pull request merged**